app/Policies/AchievementSetPolicy.php

```diff
@@ -37,32 +34,27 @@ public function create(User $user): bool
     {
         return $user->hasAnyRole([
             Role::GAME_HASH_MANAGER,
-            Role::DEVELOPER,
-            Role::DEVELOPER_JUNIOR,
         ]);
     }
 
     public function update(User $user): bool
     {
         return $user->hasAnyRole([
             Role::GAME_HASH_MANAGER,
-            Role::DEVELOPER,
```

Comment:
As a developer without hash manager permissions, I can still click on a row in the Game's Set list, and I get a forbidden error. I expect the page should open readonly as `view` returns true. But if it's not supposed to open, then the click shouldn't do anything.

Reply:
Disabled the row links in latest. The title labels still have an underline on hover which Filament doesn't seem to give me any way to get rid of, but this will naturally go away as a result of #4139.